### PR TITLE
FEATURE: Bundle discourse-footnote plugin into core

### PR DESCRIPTION
**Pull request by oblakeerickson**
Formerly https://github.com/discourse/discourse-footnote

### CODE REVIEW

**Comment by oblakeerickson** [comment]
I think the plugin tests won't pass until https://github.com/discourse/discourse-footnote/pull/68 is merged in.

File: plugins/footnote/translator.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Configuration file for discourse-translator-bot
+
+files:
+- source_path: config/locales/server.en.yml
+  destination_path: server.yml
```
Comment (pmusaraj) [inline plugins/footnote/translator.yml:5]: Is this file needed?  I believe the top-level `translator.yml` covers this...

File: plugins/footnote/package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "author": "Discourse",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint-config-discourse": "^3.4.0"
+  }
+}
```
Comment (pmusaraj) [inline plugins/footnote/package.json:7]: This file is probably not needed either now that footnote is in core, it should follow core's eslint config.

File: plugins/footnote/config/settings.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+plugins:
+  enable_markdown_footnotes:
+    default: true
+    client: true
+  display_footnotes_inline:
+    default: true
+    client: true
```
Comment (pmusaraj) [inline plugins/footnote/config/settings.yml:7]: I wonder if this is a problem... it'll be default on for all instances now (whereas before you'd have to install the plugin). Let's discuss internally.

**Review by pmusaraj — APPROVED** [review]
